spec/element_spec.rb

```diff
@@ -75,6 +75,20 @@
 
       expect(browser.div(:id, 'text')).to_not exist
     end
+
+    it "returns appropriate value when an ancestor element becomes stale" do
+      stale_element = browser.div(id: 'top').div(id: 'middle').div(id: 'bottom')
+      expect(stale_element.present?).to be true # look up and store @element for each element in hierarchy
+
+      grandparent = stale_element.instance_variable_get('@parent').instance_variable_get('@parent').instance_variable_get('@element')
```

Comment:
Yeah, we have 2 different things named parent.

@parent, is a private instance variable of Watir::Element denoting the element/frame/browser that is the ancestor.
Watir::Element#parent is a method that calls execute_atom :getParentElement

It would be great to name the private one differently, like ancestor or something, but I didn't want to go there for this.